test/data/javascript/tests.js

```diff
@@ -3304,6 +3293,42 @@ exports.test_data = {
             '}'
           ]
         },
+        {
+          comment: "#1838 - handle class and interface word as an object property",
+          unchanged: [
+            '{',
+            '    class: {',
+            '        a: 1,',
+            '        b: 2,',
+            '        c: 3,',
+            '    }',
+            '    interface: {',
+            '        a: 1,',
+            '        b: 2,',
+            '        c: 3,',
+            '    }',
+            '}'
+          ]
+        },
+        {
+          comment: "#1838 - handle class word as an object property but with space after colon",
+          input: [
+            '{',
+            '    class : { a: 1,',
+            'b: 2,c : 3',
+            '    }',
+            '}'
+          ],
```

Comment:
This is good.  I think you need to test at least two more inputs:
* `{class:{a:1,b:2,c:3,}}`
* `{x:{a:1,class:2,c:3,}}`